script.js

```diff
@@ -0,0 +1,9 @@
+// Display the prompt dialogue while the value assigned to `secret` is not equal to "sesame"
+let secret;
```

Comment:
I think there is no need for prompt to be assigned to this variable. You may delete these